docs/eventing/samples/sequence/sequence-with-broker-trigger/README.md

```diff
@@ -1,14 +1,15 @@
 We are going to create the following logical configuration. We create a
-PingSource, feeding events into the Broker, then we create a `Filter` that
-wires those events into a [`Sequence`](../../../flows/sequence.md) consisting of 3
+PingSource, feeding events into the Broker, then we create a `Filter` that wires
+those events into a [`Sequence`](../../../flows/sequence.md) consisting of 3
 steps. Then we take the end of the Sequence and feed newly minted events back
 into the Broker and create another Trigger which will then display those events.
```

Comment:
```suggestion
PingSource, that feeds events to the broker, then create a filter that wires
those events into a [sequence](../../../flows/sequence.md) consisting of 3
steps. Then take the end of the sequence, and feed the newly created events back
into the broker, and create another trigger which will then display those events.
```

Reply:
This is a very long and confusing list to be written this way. It would be better as a numbered list of steps IMHO.